app/api/tests/test_userview.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import json
+from app.api.tests.test_base import BaseTest
+
+
+class TestUser(BaseTest):
+	"""docstring for TestUser"""
+	def test_create_user(self):
+		respo= self.client.post('/api/v1/users/signup',data = json.dumps(self.user_dummy_data),
+								content_type='application/json')
```

Comment:
Since this is sign up, use `self.signup_dummy_data` instead. Ie, `data = json.dumps(self.signup_dummy_data)`